Python/Product/Analysis/Interpreter/PythonInterpreterFactoryExtensions.cs

```diff
@@ -56,12 +56,6 @@ public static class PythonInterpreterFactoryExtensions {
         /// <returns>The names of the modules that were found.</returns>
         public static async Task<HashSet<string>> FindModulesAsync(this IPythonInterpreterFactory factory, params string[] moduleNames) {
             var withDb = factory as PythonInterpreterFactoryWithDatabase;
-            if (withDb != null && withDb.IsCurrent) {

```

Comment:
Some of my branches got cross merged because I was waiting so long for reviews :)